rotors_hil_interface/include/rotors_hil_interface/hil_listeners.h

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2016 Pavel Vechersky, ASL, ETH Zurich, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROTORS_HIL_LISTENERS_H_
+#define ROTORS_HIL_LISTENERS_H_
+
+#include <boost/thread/mutex.hpp>
+#include <Eigen/Dense>
+#include <geometry_msgs/TwistStamped.h>
+#include <ros/ros.h>
+#include <sensor_msgs/FluidPressure.h>
+#include <sensor_msgs/Imu.h>
+#include <sensor_msgs/MagneticField.h>
+#include <sensor_msgs/NavSatFix.h>
+
+namespace rotors_hil {
+// Constants
+static constexpr float kAirDensity = 1.18;
+static constexpr float kGravityMagnitude = 9.8068;
```

Comment:
Is this the only place where these constants are defined? Just to avoid having e.g. different gravity values for the same simulation session..